.github/workflows/performance-benchmarks-bayesmark.yml

```diff
@@ -105,7 +105,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install --progress-bar off -U setuptools wheel
-        pip install --progress-bar off numpy scipy pandas
+        pip install --progress-bar off numpy scipy pandas Jinja2
```

Comment:
Should we update `How to run locally` section as well?
https://github.com/optuna/optuna/tree/master/benchmarks#how-to-run-locally-1

Reply:
Thanks, updated in 50ca9f23a81078df1004d9ef496e09d9527a324a! On the related note - I've planned a bigger update of bayesmark README section as separate work piece to follow sometime soon since report and scoring logic evolved a bit.